### ISSUE TITLE
Getting net::ERR_ABORTED 429 from YouTube iframe

### ISSUE THREAD

**Issue by windevkay** [CookPete/react-player#780]
#### Current Behavior
`GET https://www.google.com/sorry/index?continue=https://www.youtube.com/iframe_api&q=EgTQYsTCGL_RgvEFIhkA8aeDSyLhUWkq-8ef3f0lqFHuXs9f6QB-MgFy net::ERR_ABORTED 429`

The player has been working fine but suddenly started getting this issue. The 429 status seems to indicate an excess of requests to the iframe API and i'm not sure if there is a workaround for this or something is wrong with my code

#### Expected Behavior
Once onReady prop is fired, callback should place a scroll event listener and once a user scrolls on the page, playing variable in state is set.

#### Steps to Reproduce
```
import React from 'react';
import YouTubePlayer from 'react-player/lib/players/YouTube';
import { PlayerWrapper, MediaWrapper } from './styles';

class VideoPlayer extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      playing: false,
      isMounted: false
    }
  }

  componentDidMount() {
    this.setState({ isMounted: true });
  }

  componentWillUnmount() {
    this.setState({ isMounted: false });
    this.detachEventListener();
  }

  attachEventListener = () => {
    window.addEventListener('scroll', this.setPlaying);
  }

  detachEventListener = () => {
    window.removeEventListener('scroll', this.setPlaying);
  }

  setPlaying = () => {
    this.setState({ playing: true })
  }

  render() {
    const { playing } = this.state;
    return (
      <MediaWrapper>
        <PlayerWrapper>
          <YouTubePlayer
            className='react-player'
            url='https://www.youtube.com/watch?v=vRJOn0g0CEc'
            onReady={this.attachEventListener}
            playing={playing} width={'100%'} height={'100%'}
            controls={true}
          />
        </PlayerWrapper>
      </MediaWrapper>
    );
  }
}

export default VideoPlayer;
```

#### Environment
- URL attempting to play: https://www.youtube.com/watch?v=vRJOn0g0CEc
- Browser: Google Chrome
- Operating system: Windows 10 Pro
- jsFiddle example: 
[player.txt](https://github.com/CookPete/react-player/files/4073030/player.txt)

#### Other Information
FYI I get that same console error even when I copy the direct code sample from NPM for this package and try to run it.

**Comment by CookPete** [comment]
Possibly useful links:

- https://support.google.com/youtube/thread/21939228?hl=en
- https://stackoverflow.com/questions/40756171/youtube-reports-api-httperror-429-freequotagroupclient-project-100
- https://github.com/ytdl-org/youtube-dl/issues/21729
- https://github.com/ytdl-org/youtube-dl/issues/22839

TLDR: Looks like it's a problem with the server/domain you're hosting your app on and the amount of times it's hitting youtube.

**Comment by TonyKim0627** [comment]
@windevkay windevkay were you able to resolve this issue?

**Comment by windevkay** [comment]
@TonyKim0627 nope. The package wasn't the issue though, it was from YouTube. By the next day everything was working fine again.
